--- cosas/cosas.py.org	2012-09-26 17:35:34.000000000 +0000
+++ cosas/cosas.py	2016-01-23 13:36:58.425223509 +0000
@@ -20,8 +20,13 @@
 import re
 import u1db
 import copy
+import platform
 
-from dirspec.basedir import save_data_path
+if platform.system() != 'Windows':
+  from dirspec.basedir import save_data_path
+else:
+  def save_data_path():
+    return os.path.join(os.environ["APPDATA"], "u1db")
 
 
 TAGS_INDEX = 'tags'
