--- setup.py.org	2012-09-26 17:35:34.000000000 +0000
+++ setup.py	2016-01-23 13:29:37.857043141 +0000
@@ -40,7 +40,7 @@
         "package_data": {'': ["*.sql"]},
         "scripts": ['u1db-client', 'u1db-serve'],
         "ext_modules": ext,
-        "install_requires": ["dirspec", "paste", "routes"],
+        "install_requires": ["paste", "routes"],
         # informational
         "tests_require": ["testtools", "testscenarios", "Cython",
                           "pyOpenSSL"],
